pkg/utils/util.go

```diff
@@ -56,3 +59,57 @@ func GetJSON(client *http.Client, url string, v interface{}) error {
 
 	return errors.Trace(json.NewDecoder(resp.Body).Decode(v))
 }
+
+// GetGlobalVariable gets server's global variable
```

Comment:
these functions copied from DM https://github.com/pingcap/dm/blob/56eed525e047f1ecf1ce03709eb5ccb1e7b50caf/pkg/utils/db.go#L356

Reply:
How about putting them into _dbutil_ rather than _utils_?

Reply:
done in https://github.com/pingcap/tidb-tools/pull/381/commits/efda573e693caa32aea1e3dfd43e1e70725bf750